data/transactions/logic/TEAL_opcodes.md

```diff
@@ -464,7 +457,7 @@ FirstValidTime causes the program to fail. The field is reserved for future use.
 
 - Opcode: 0x33 {uint8 transaction group index} {uint8 transaction field index}
 - Stack: ... &rarr; ..., any
-- field F of the Tth transaction in the current group
+- gtxn can be called using gtxn with 2 immediates. field F of the Tth transaction in the current group
```

Comment:
So in this case, I agree appending would be better, but at the time it seemed like prepending was best since it's pretty crucial info that I'd rather not see at the end of some long description.

For now, I'll switch it to append and then in the future, we can reevaluate as needed.